### PR TITLE
Installation: TBBROOT -> TBB_ROOT

### PR DESCRIPTION
**Pull request by maxGimeno**
## Summary of Changes

Update the documentation and the CHANGES.md files for the new TBB_ROOT variable.

## Release Management

* Affected package(s):Installation

### CODE REVIEW

File: Installation/CHANGES.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ Release 5.0
 
 Release date: September 2019
 
+### Installation
+-   The FindTBB.cmake file has been updated to work with later versions of TBB. The TBBROOT variable now becomes TBB_ROOT.
```
Comment (lrineau) [inline Installation/CHANGES.md]:
```suggestion
-   The `FindTBB.cmake` file has been updated to work with later versions of TBB. The `TBBROOT` environment variable now becomes `TBB_ROOT`.
```

File: Documentation/doc/Documentation/Installation.txt [hunk 2]
```diff
@@ -1144,18 +1144,18 @@ environment variable is sufficient.
...
+For windows users, the folder `TBB_ROOT/bin/<arch>/<compiler>` should be added to the `PATH` variable.
 
 Note that the variables in the table below are being used.
 
 | Variable             | Description           | Type        |
 | :-                   | :-                    | :-             |
-| `TBBROOT` | Directory of \sc{Tbb} default installation | Environment |
+| `TBB_ROOT` | Directory of \sc{Tbb} default installation | Environment |
 | `TBB_INCLUDE_DIRS` | Directory containing the `tbb/tbb.h` file | CMake |
 | `TBB_LIBRARY_DIRS` | Directory(ies) containing the compiled TBB libraries | CMake |
```
Comment (lrineau) [inline Documentation/doc/Documentation/Installation.txt]: `TBB_LIBRARY_DIRS` is not set or used by the new `FindTBB` module.

**Comment by lrineau** [comment]
This pull-request will probably get cancelled, and replace by #4252.

**Comment by maxGimeno** [comment]
If #4252 is intergrated, this PR won't have any sense and should be closed without merging.

**Comment by lrineau** [comment]
#4252 is now merged, and reintroduce the use of `TBBROOT`. I close this one.

**Pull request closed without merging**